upup/pkg/fi/cloudup/awstasks/network_load_balancer.go

```diff
@@ -365,6 +366,34 @@ func (e *NetworkLoadBalancer) Find(c *fi.Context) (*NetworkLoadBalancer, error)
 					return nil, err
 				}
 				actual.CrossZoneLoadBalancing = fi.Bool(b)
+			case "access_logs.s3.enabled":
+				b, err := strconv.ParseBool(*value)
+				if err != nil {
+					return nil, err
+				}
+				if actual.AccessLog != nil {
+					actual.AccessLog.Enabled = fi.Bool(b)
+				} else {
+					actual.AccessLog = &NetworkLoadBalancerAccessLog{
+						Enabled: fi.Bool(b),
+					}
+				}
```

Comment:
These could probably be rewritten as:

```suggestion
				if actual.AccessLog == nil {
					actual.AccessLog = &NetworkLoadBalancerAccessLog{}
				}
				actual.AccessLog.Enabled = fi.Bool(b)
```